include/spo2_filter.h

```diff
@@ -0,0 +1,23 @@
+#ifndef __SPO2_FILTER__
+#define __SPO2_FILTER__
+
+#include "spo2.h"
+#include "spo2_driver.h"
+
+#define ADC_OFFSET_HALF 2048
+
+#define DC_LOWPASS(fb,adc) ((((126*fb)>>7)+((2*(adc))>>7)))
```

Comment:
Does these two need to be macros?
Please have a look at [6.3 a and b](https://barrgroup.com/embedded-systems/books/embedded-c-coding-standard/procedure-rules/preprocessor-macros)

Reply:
I will try to automate checks in near future

Reply:
Applied as requested.